packages/react-components/theme-designer/src/components/Export/ExportPanel.tsx

```diff
@@ -116,67 +182,15 @@ export const ExportPanel = () => {
     <>
       {showExportPanel && (
         <FluentProvider theme={webLightTheme}>
-          <div
-            style={{
-              zIndex: 100,
-              position: 'absolute',
-              top: '0px',
-              right: '0px',
-              width: '400px',
-              border: `1px solid ${tokens.colorNeutralStroke1}`,
-              borderRadius: tokens.borderRadiusXLarge,
-              backgroundColor: tokens.colorNeutralBackground1,
-              boxShadow: tokens.shadow64,
-            }}
-          >
-            <div style={{ margin: '16px' }}>
-              <div className={styles.exportHeader}>
-                <Text as="h1" id="headingID" size={500}>
-                  Export Theme
-                </Text>
-                <Button
-                  size="small"
-                  appearance="subtle"
-                  icon={<DismissSquare24Regular />}
-                  // eslint-disable-next-line react/jsx-no-bind
-                  onClick={onCloseExportPanel}
-                />
-              </div>
-
-              <br />
-              <Body1>
-                Passing this theme to a FluentProvider will automatically apply it to any Fluent components below it.
-                You can also export this to CodeSandbox with a few component examples below.
-              </Body1>
-              <br />
-              <TabList
-                defaultSelectedValue="Code"
-                selectedValue={selectedValue}
-                onTabSelect={onTabSelect} // eslint-disable-line react/jsx-no-bind
-              >
-                <Tab value="Code">Code</Tab>
-                <Tab value="JSONLight">JSON (light)</Tab>
-                <Tab value="JSONDark">JSON (dark)</Tab>
-              </TabList>
-              <Textarea
-                className={styles.text}
-                size="small"
-                value={exportedValue}
-                id={'textArea'}
-                textarea={{ className: styles.textarea }}
-                readOnly
-              />
-              <br />
-              <ExportLink />
-              <br />
-              <Button
-                appearance="primary"
-                // eslint-disable-next-line react/jsx-no-bind
-                onClick={onClickCopyToClipboard}
-              >
-                Copy to clipboard
-              </Button>
-            </div>
+          <div className={styles.panelContainer}>
+            <ExportPanelHeader onClose={onCloseExportPanel} />
+            <br />
```

Comment:
I know this is not your code, but we should remove random line breaks. (nyc)

They're often improperly used by people unfamiliar with CSS layout techniques to give margin or spacing.

They really should only be used in long form, paragraph text content.